WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.m

```diff
@@ -162,6 +198,15 @@ - (WPXMLRPCRequestOperation *)operationForPostFormatsWithSuccess:(PostFormatsHan
     return operation;
 }
 
+- (RemotePostType *)remotePostTypeFromXMLRPCDictionary:(NSDictionary *)json
+{
+    RemotePostType *postType = [[RemotePostType alloc] init];
```

Comment:
See previous regarding a initializer that accepts params for non null properties.